data/config.dist.json

```diff
@@ -12,5 +12,6 @@
     "Reflection",
     "SPL",
     "standard"
-  ]
+  ],
+  "scan-files" : []
```

Comment:
Would it make sense for us to scan files according to the composer autoloader definitions?

Reply:
Well, we already scan files depending on `autoload` but in this case it could be possible that a file is not within the autoload definition, because it's a simple executable script living in another directory, as mentioned in #76